compile

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 compass compile --sass-dir src/style/ --css-dir ./
```

Comment:
I'm not sure how this line works - I tried running `npm i` but received an error saying `compass is not found`, and it doesn't look like the library exposes a command so even installing it globally shouldn't work.

Reply:
oh yeah, IIRC I did actually have to install compass globally.  It's a Ruby gem, so the executable is in the gems directory, which is already on my PATH after updating Ruby.

I'll add instructions to the README.